docs/rgbasm.5.html

```diff
@@ -1303,7 +1303,7 @@ <h2 class="Ss" title="Ss" id="String_Expressions"><a class="selflink" href="#Str
   </tr>
 </table>
 <div class="Pp"></div>
-A funky feature is <b class="Sy" title="Sy">{symbol}</b> withing a string. This
+A funky feature is <b class="Sy" title="Sy">{symbol}</b> within a string. This
```

Comment:
We need to make this clearer, but the HTML docs are auto-generated from the man docs. This particular file is generated from `src/rgbds.5`, so if you could fix the typo there, that’d be much appreciated!

Reply:
Hmm... 
In "src/rgbds.5", there is no word "watching".
In "src/asm/rgbasm.5" There is a word "watching".
So. I fix src/asm/rgbasm.5 file.

Reply:
Whoops, I was skim-reading and misread this filename! Thanks 😄